src/components/include/application_manager/hmi_capabilities.h

```diff
@@ -521,6 +535,18 @@ class HMICapabilities {
    */
   virtual std::set<hmi_apis::FunctionID::eType>
   GetDefaultInitializedCapabilities() const = 0;
+
+  /**
+   * @brief Notified that response was received from requested interface
+   * @param requested_interface interface for which received response
+   */
+  virtual void InterfaceResponseReceived(
+      hmi_apis::FunctionID::eType requested_interface) = 0;
+
+  /**
+   * @brief Init interfaces for which should wait for the responses
```

Comment:
@ydementieiev For which interfaces we should wait for responses?

Reply:
@AByzhynar i deleted this function